app/controllers/UserController.js

```diff
@@ -34,6 +35,54 @@ UserController.prototype.getUsersByOrg = function () {
   }
 };
 
+UserController.prototype.create = function ()
+{
+  var opts = {
+    on: {
+      preCreate: function (req, doc, cb) {
+        async.waterfall ([
+          function (callback) {
+            var token = req.headers.authorization.split(' ')[1];
+
+            User.findOne ({token: token}, function (err, admin) {
+              if (err) { callback (err); }
+
+              doc.org_id = admin.org_id;
+              callback (null, doc);
+            });
+          },
+
+          function (doc, callback) {
+            User.findOne ({email: doc.email}, function (err, user) {
+              if (err) { callback (err); }
```

Comment:
though I feel it probably won't happen, if both user and err are sent call back will be called twice since there is not a return statement